src/lib/helpers.js

```diff
@@ -114,6 +114,40 @@ export function createHelpers({ cache, webAPI }) {
       return data
     },
 
+    async externalRefs(extRef) {
```

Comment:
No need to be async. I feel a map will be more efficient and would keep the mappings in the serializers. e.g.

```
function externalRefs(extRefs) {
  return extRefs.reduce((map, ref) => {
    map.set(ref.type, ref.identifier);
    return map;
  }, new Map());
}

const refsMap = externalRefs(extRefs);

console.log(refsMap.get('loggernet.data_tables'));
```